cls/SourceControl/Git/Utils.cls

```diff
@@ -2798,4 +2798,28 @@ ClassMethod BaselineExport(pCommitMessage = "", pPushToRemote = "") As %Status
     return sc
 }
 
+/// Returns the url for the remote repository (censoring the username)
+ClassMethod GetConfiguredRemote() As %String
+{
+    d ..RunGitCommand("remote",.err,.out,"-v")
+    set line = out.ReadLine()
+    set url = $piece($piece(line,$char(9),2)," ",1)
+    return url
+}
+
+ClassMethod GetCensoredRemote() As %String
```

Comment:
I'm going to be pedantic and say we should call this "GetRedactedRemote" instead, here...